app-modules/group/src/Policies/GroupPolicy.php

```diff
@@ -60,83 +60,83 @@ public function viewAny(Authenticatable $authenticatable): Response
         }
 
-        if ($segment->campaigns()->exists()) {
-            return Response::deny('This population segment is associated with a campaign. Please remove the campaign before attempting to remove the segment.');
+        if ($group->campaigns()->exists()) {
+            return Response::deny('This population group is associated with a campaign. Please remove the campaign before attempting to remove the group.');
         }
 
         return $authenticatable->canOrElse(
-            abilities: ["segment.{$segment->getKey()}.delete"],
-            denyResponse: 'You do not have permission to delete this segment.'
+            abilities: ["segment.{$group->getKey()}.delete"],
+            denyResponse: 'You do not have permission to delete this group.'
         );
     }
 
-    public function restore(Authenticatable $authenticatable, Segment $segment): Response
+    public function restore(Authenticatable $authenticatable, Group $group): Response
     {
-        if (! $authenticatable->hasLicense($segment->model?->class()::getLicenseType())) {
-            return Response::deny('You do not have permission to restore this segment.');
+        if (! $authenticatable->hasLicense($group->model->class()::getLicenseType())) {
+            return Response::deny('You do not have permission to restore this group.');
         }
 
         return $authenticatable->canOrElse(
-            abilities: ["segment.{$segment->getKey()}.restore"],
-            denyResponse: 'You do not have permission to restore this segment.'
+            abilities: ["segment.{$group->getKey()}.restore"],
+            denyResponse: 'You do not have permission to restore this group.'
         );
     }
 
-    public function forceDelete(Authenticatable $authenticatable, Segment $segment): Response
+    public function forceDelete(Authenticatable $authenticatable, Group $group): Response
     {
-        if (! $authenticatable->hasLicense($segment->model?->class()::getLicenseType())) {
-            return Response::deny('You do not have permission to permanently delete this segment.');
+        if (! $authenticatable->hasLicense($group->model->class()::getLicenseType())) {
+            return Response::deny('You do not have permission to permanently delete this group.');
         }
 
-        if ($segment->campaigns()->exists()) {
-            return Response::deny('This population segment is associated with a campaign. Please remove the campaign before attempting to remove the segment.');
+        if ($group->campaigns()->exists()) {
+            return Response::deny('This population group is associated with a campaign. Please remove the campaign before attempting to remove the group.');
         }
 
         return $authenticatable->canOrElse(
-            abilities: ["segment.{$segment->getKey()}.force-delete"],
-            denyResponse: 'You do not have permission to permanently delete this segment.'
+            abilities: ["segment.{$group->getKey()}.force-delete"],
```

Comment:
Since we are in this file now, please change this to the `*` version of the permission check, don't pass the key